index.html

```diff
@@ -1085,6 +924,213 @@ <h3>Implicit Release of Pointer Capture</h3>
+                coordinates from <code>pointermove</code> events (the grey dots), the curve is noticeably
+                angular and jagged; the same line drawn using the more granular points provided by
+                <code>getCoalescedEvents</code> (the red circles) results in a smoother approximation
+                of the pointer movement.</figcaption>
+            </figure>
+
+            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced event list</dfn> (a list of
+            zero or more <code>PointerEvents</code>). If this event is a <code>pointermove</code>
+            or <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>
+            that were coalesced into this event; otherwise it is an empty list.</p>
+
+            <p>The events in the coalesced event list will have increasing
+            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
+            ([[!WHATWG-DOM]]), so the first event will have the smallest <code>timeStamp</code>.</p>
+
+            <pre id="example_10" class="example" title="Basic canvas drawing application used the coalesced events list">
+                <code>&lt;style&gt;
+                    /* Disable intrinsic user agent touch behaviors (such as panning or zooming) so
+                    that all events on the canvas element are given to the application instead. */
+                
+                    canvas { touch-action: none; }
+                &lt;/style&gt;
+                
+                &lt;canvas id="drawSurface" width="500px" height="500px" style="border:1px solid black;"&gt;&lt;/canvas&gt;
+                
+                &lt;script&gt;
+                    var canvas = document.getElementById("drawSurface"),
+                    context = canvas.getContext("2d");
+        
+                    canvas.addEventListener("pointermove", (e)=> {
+
+                        if (e.getCoalescedEvents) {
+                            for (let coalesced_event of e.getCoalescedEvents()) {
+                                paint(coalesced_event); // Paint all raw/non-coalesced points
+                            }
+                        } else {
+                            paint(e); // Paint the final coalesced point
+                        }
+                    });
+                
+                    function paint(event) {
+                        if(event.buttons>0) {
+                            context.fillRect(event.clientX, event.clientY, 5, 5);
+                        }
+                    }
+                
+                &lt;/script&gt;</code>
+            </pre>
+            
+            <div class="note">The PointerEvent's attributes will be initalized in a way that best represents
+            all events in the coalesced event list.
+            For example its <a href="https://www.w3.org/TR/pointerlock/#widl-MouseEvent-movementX">
+            movementX</a> and
```

Comment:
For non-movement, shouldn't coordinates be the most recent ones, not "best represents
all events in the coalesced event list"

Reply:
i wonder if this is an implementation detail? i.e. could user agents choose to do coalescing in terms of averaging, rather than just picking the last/most recent one? do any do that?

Reply:
Made a tiny tweak changing "all events" to "the events". Taking a step back though, I'm actually unclear on what this note is trying to say. Right from the start, "The PointerEvent's attributes" ... which pointer event is it referring to? The coalesced "parent" pointer event that has the list of non-coalesced pointer events? If so, is this trying to define (in a non-normative note, but still) what a user agent could do? The spec already says that it isn't trying to define this. Wonder if the whole note should not be removed? I think it's worth opening a new issue for this though, as this question is essentially about existing text (rather than about the reshuffling around of that into a new section)